packages/core/stories/dropdown/dropdown.stories.tsx

```diff
@@ -208,7 +206,7 @@ export const ComplexOption: StoryFn<DropdownNextProps> = (args) => {
   const adornment = countries[selected[0] ?? ""]?.icon || null;
 
   return (
-    <DropdownNext
+    <Dropdown
```

Comment:
the start adornment seems a bit too to the right compared to input adornments, should the DD startadornment icons have a no margin style? or is that up to the user?

Reply:
Dropdown seemed fine, ComboBox was out